[Command Bug] New adventure install issues w/ character info

## Complete these first
- [ ] I confirm that I am the owner of the bot running Adventure.
- [x] I confirm that I have not modified the source code of Adventure.
- [ ] I'm an Adventure user but not the bot owner.
- [x] I confirm that I have checked the [issue tracker](../) and a this bug hasn't been reported already.

## Command name

stats, give item, things that touch the character sheet

## Expected Behavior

The commands should work

## Current Behavior
3.4.3.2
New bot/new adventure install

running stats:
```
[2021-01-15 14:19:40] [ERROR] red.cogs.adventure: Error with the new character sheet
Traceback (most recent call last):
  File "/home/ubuntu/.local/share/Red-DiscordBot/data/yuri/cogs/CogManager/cogs/adventure/adventure.py", line 5058, in stats
    c = await Character.from_json(self.config, user, self._daily_bonus)
  File "/home/ubuntu/.local/share/Red-DiscordBot/data/yuri/cogs/CogManager/cogs/adventure/charsheet.py", line 1464, in from_json
    balance = await bank.get_balance(user)
  File "/home/ubuntu/.local/share/Red-DiscordBot/data/yuri/cogs/CogManager/cogs/adventure/bank.py", line 135, in get_balance
    acc = await get_account(member, _forced=_forced)
  File "/home/ubuntu/.local/share/Red-DiscordBot/data/yuri/cogs/CogManager/cogs/adventure/bank.py", line 487, in get_account
    return await bank.get_account(member)
  File "/home/ubuntu/yuri/lib/python3.8/site-packages/redbot/core/bank.py", line 621, in get_account
    all_accounts = await _config.all_members(member.guild)
AttributeError: 'User' object has no attribute 'guild'
```

tracebacks from 2 users at:
https://discord.com/channels/240154543684321280/735324427302076456/799658477046857778
https://discord.com/channels/240154543684321280/735324427302076456/813994740835287042

## Possible Solution
Time to investigate.

## Steps to Reproduce
1. Get a new Red, install adventure
2. Try stats on people
3. Give an item to someone
4. Try stats on another person

## Context (Environment)
One user's info at https://discord.com/channels/240154543684321280/735324427302076456/814317795910746133

## Detailed Description


## Possible Implementation
Seems like the custom bank file is expecting a Member where it's being passed a User instead.

--- a/adventure/charsheet.py
+++ b/adventure/charsheet.py
@@ -1299,7 +1299,10 @@ class Character:
     ):
         """Return a Character object from config and user."""
         data = await config.user(user).all()
-        balance = await bank.get_balance(user)
+        try:
+            balance = await bank.get_balance(user)
+        except Exception:
+            balance = 0
         equipment = {k: Item.from_json(ctx, v) if v else None for k, v in data["items"].items() if k != "backpack"}
         if "int" not in data["skill"]:
             data["skill"]["int"] = 0

